client/apps/Admin/src/pages/AssignmentsPage.tsx

```
@@ -0,0 +1,453 @@
+
+                return (
+                    <div className="assignments-page__actions">
+                        {isEditing ? (
+                            <button
+                                className="assignments-page__action-btn assignments-page__action-btn--active"
+                                onClick={() => setEditingAssignmentsTaskId(null)}
+                                title="סיום עריכה"
+                            >
+                                סיום
+                            </button>
+                        ) : (
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)